modules/organizational-unit/main.tf

```diff
@@ -1,14 +1,14 @@
 /**
  * # AWS Organizational Unit terraform module
  *
- * This module creates AWS Organization OUs and uses AWS Accounts module to 
+ * This module creates AWS Organization OUs and uses AWS Accounts module to
  * create accounts inside of that OU.
  *
  * It also provides functionality to attach a list of organizational policies to the OU.
  */
 
 
-resource "aws_organizations_organizational_unit" "this_orgranizations_organizational_unit" {
+resource "aws_organizations_organizational_unit" "this_organizations_organizational_unit" {
```

Comment:
Unfortunately changing that names (spelling error) is valid but causes a breaking change and will introduce problems with existing terraform state